docs/src/extensions_folder/h5browser.rst

```diff
@@ -39,3 +42,35 @@ Some options are available when right clicking on a node, see :numref:`figure_h5
 * Plot Nodes: plot data hanging from the same channel
 * Plot Node with Bkg: plot data with subtracted background (if present)
 * Plot Nodes with Bkg: plot data hanging from the same channel with subtracted background (if present)
+
+Associating H5Browser with .h5 files
++++++++++++++++++++++++++++++++++++++
+
+By default, the H5Browser always asks the user to select a file. One can instead open a specified .h5 file directly,
+using the --input optional command line argument as follows:
+
+``h5browser --input my_h5_file.h5``.
+
+One can also associate H5Browser to all .h5 file so that it directly opens a file when double clicking on it. Here is
+how to do it on Windows. Let us assume that you have a conda environment named *my_env*, in which PyMoDAQ is installed.
+
+In Windows, the path to your conda executable will be something like:
+
+``C:\Miniconda3\condabin\conda.bat``
+
+and the path to the H5Browser will be (note that we are going into the *my_env* folder):
+
+``C:\Miniconda3\envs\my_env\Lib\site-packages\pymodaq\h5browser.py``
+
+Now that you have written down these two paths, open your favorite text editing tool (e.g. notepad) and create a file
+called *H5Opener.bat* (for instance) with the following contents:
+
+  .. code-block:: python
+
+    @ECHO OFF
+    call C:\Miniconda\condabin\conda.bat activate my_env
```

Comment:
missing the 3 in "Miniconda3" compared to above (sneaky!!)
perhaps add a note saying to precisely check the path

Reply:
done